site/src/components/EmptyState/EmptyState.tsx

```diff
@@ -40,24 +39,21 @@ export const EmptyState: FC<React.PropsWithChildren<EmptyStateProps>> = (
       }}
       {...boxProps}
     >
-      <Typography variant="h5" css={{ fontSize: 24 }}>
```

Comment:
I would expect an `<h5>` to at least have a weight of 500

Reply:
I agree, I think this could be something we should set as a global style? Wondering if as part of the theme thing, we could include the TailwindCSS reset.

Reply:
but my point was that _here_, you manually set it to 400. it should be a `<p>` or something instead, not a header.

Reply:
Ahhh ok, but it is a title so I think makes sense to make it a heading element 🤔